src/main/main.ts

```diff
@@ -152,13 +154,18 @@ const createWindow = async () => {
         : path.join(__dirname, '../../.erb/dll/preload.js'),
     },
   });
-
+  
   mainWindow.loadURL(resolveHtmlPath('mainWindow.index.html'));
 
   mainWindow.on('ready-to-show', () => {
     if (!mainWindow) throw new Error('"mainWindow" is not defined');
 
-    updateStatus(checkConfig() ? 0 : 2);
+    const initialStatus = checkConfig() ? AppStatus.WaitingForWoW : AppStatus.InvalidConfig;
+
+    updateStatus(initialStatus);
+
+    // This shows the correct version on a release build, not during development.
+    mainWindow.webContents.send('updateTitleBar', 'Warcraft Recorder v' + app.getVersion());
```

Comment:
Where does the 18.0.3 come from I see on the dev version?

Reply:
app.getVersion() changes return value depending on how it's launched, see [here](https://github.com/electron/electron/issues/7085#issuecomment-244584704).

package.json:192 means any version of electron that is 18.x.x, so that's where 18.0.3 is coming from.
`"electron": "^18.0.1",`

Reply:
Also caught up

Reply:
Diff still not quite right, but I'll try to make do